src/Main.java

```diff
@@ -0,0 +1,400 @@
+import java.util.Scanner;
+import java.util.Random;
+public class Main {
+    public static void main(String[] args) throws InterruptedException {
+        Scanner playerInput = new Scanner(System.in);
+        Random generator = new Random();
+        Character player = new Character();
+        Character zombie = new Character();
+        Character ghost = new Character();
+        Character goblin = new Character();
+        Character dungeonBoss = new Character();
+
+        String[] monsters = {"Zombie", "Ghost", "Goblin"};
+        int rMonster = generator.nextInt(monsters.length);
+        String randomMonster = monsters[rMonster];
+
+        String[] monstersRepeatable = {"Zombie", "Ghost", "Goblin"};
+        int rMonsterRepeatable = generator.nextInt(monstersRepeatable.length);
+        String randomMonsterRepeatable = monstersRepeatable[rMonsterRepeatable];
+
+
+        int monsterAttack = generator.nextInt(6);
+        if (monsterAttack == 0) {
+            monsterAttack = 1;
+        }
+        int monsterHealth = generator.nextInt(16);
+        if (monsterHealth == 0) {
+            monsterHealth = 1;
+        }
+
+        int playerRecover1 = generator.nextInt(11);
+        if (playerRecover1 == 0) {
+            playerRecover1 = 1;
+        }
+        int playerRecover2 = generator.nextInt(6);
+        if (playerRecover2 == 0) {
+            playerRecover2 = 1;
+        }
+        int playerStrength = generator.nextInt(6);
+        if (playerStrength == 0) {
+            playerStrength = 1;
+        }
```

Comment:
If statements are generally inefficient and are to be avoided. Instead, consider doing the following `random.nextInt(max - min) + min;`. Why do you think this works?